isis/src/base/objs/FitsToJson/FitsToJson.cpp

```diff
@@ -0,0 +1,189 @@
+/** This is free and unencumbered software released into the public domain.
+The authors of ISIS do not claim copyright on the contents of this file.
+For more details about the LICENSE terms and the AUTHORS, you will
+find files of those names at the top level of this repository. **/
+
+/* SPDX-License-Identifier: CC0-1.0 */
+#include "FitsToJson.h"
+
+#include <nlohmann/json.hpp>
+#include <QString>
+
+#include "IException.h"
+#include "IString.h"
+#include "PvlObject.h"
+#include "PvlGroup.h"
+#include "PvlToJSON.h"
+
+using json = nlohmann::json;
+
+namespace Isis {
+
+  /**
+   * Convert the contents of a stream to a JSON object.
+   *
+   * @param the stream to convert
+   * @return @b json The contents of the stream as a JSON object
+   */
+  json fitsToJson(std::ifstream &fileStream) {
```

Comment:
Could we add a small comment in the description that this logic was taken from processimportfits?